homeassistant/components/laundrify/binary_sensor.py

```diff
@@ -44,8 +44,7 @@ class LaundrifyPowerPlug(
     _attr_device_class = BinarySensorDeviceClass.RUNNING
     _attr_unique_id: str
     _attr_has_entity_name = True
-    _attr_name = None
-    _attr_translation_key = "laundrify_power_plug"
+    _attr_translation_key = "wash_cycle"
```

Comment:
Unrelated change?

Reply:
Not setting the `name` property to `None` is actually related to this PR, since the `BinarySensor` is now not the main feature anymore.

As the [docs](https://developers.home-assistant.io/docs/core/entity#has_entity_name-true-mandatory-for-new-integrations) say:
> If the entity represents a single main feature of a device the entity should typically have its name property return None.

However, changing the `translation_key` is indeed unrelated and I'd aggree to move that to a separate PR if you want me to.

Reply:
How come its not the main future anymore?

Reply:
The `BinarySensor` is no longer the main feature because this PR adds the `Sensor` platform, which is equally important.

Reply:
Let's keep this in a separate PR

Reply:
May I ask you to provide a reason for your suggestion?

I am aware about the concept of [Make your pull request as small as possible](https://developers.home-assistant.io/docs/creating_component_code_review/#5-make-your-pull-request-as-small-as-possible), but I think removing the `name` property of the `BinarySensor` is directly related to this PR and should therefore be included in here.

Please note that the change of the `translation_key` has already been moved to a separate PR (https://github.com/home-assistant/core/pull/121435).

Reply:
But if its already merged I am not going to lay in the way